#include <stdexcept>
#include <iostream>
#include <vector>

int main(){
	std::exception e;
	return 0;
}
